package.json

```diff
@@ -110,39 +107,39 @@
   },
   "dependencies": {
     "@types/parse-json": "^4.0.0",
-    "import-fresh": "^3.1.0",
+    "import-fresh": "^3.2.1",
     "parse-json": "^5.0.0",
     "path-type": "^4.0.0",
-    "yaml": "^1.7.2"
+    "yaml": "^1.10.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-typescript": "^7.6.0",
-    "@types/jest": "^24.0.19",
-    "@types/node": "^12.11.5",
-    "@typescript-eslint/eslint-plugin": "^2.5.0",
-    "@typescript-eslint/parser": "^2.5.0",
-    "cross-env": "^6.0.3",
+    "@babel/cli": "^7.10.4",
+    "@babel/core": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
+    "@babel/preset-typescript": "^7.10.4",
+    "@types/jest": "^26.0.4",
+    "@types/node": "^14.0.22",
+    "@typescript-eslint/eslint-plugin": "^3.6.0",
+    "@typescript-eslint/parser": "^3.6.0",
+    "cross-env": "^7.0.2",
     "del": "^5.1.0",
-    "del-cli": "^3.0.0",
-    "eslint": "^6.5.1",
+    "del-cli": "^3.0.1",
+    "eslint": "^7.4.0",
     "eslint-config-davidtheclark-node": "^0.2.2",
-    "eslint-config-prettier": "^6.4.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jest": "^22.20.0",
-    "eslint-plugin-node": "^10.0.0",
-    "husky": "^3.0.9",
-    "jest": "^24.9.0",
-    "lint-staged": "^9.4.2",
-    "make-dir": "^3.0.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jest": "^23.18.0",
+    "eslint-plugin-node": "^11.1.0",
+    "husky": "^4.2.5",
+    "jest": "^26.1.0",
+    "lint-staged": "^10.2.11",
+    "make-dir": "^3.1.0",
     "parent-module": "^2.0.0",
-    "prettier": "^1.18.2",
+    "prettier": "^2.0.5",
     "remark-preset-davidtheclark": "^0.12.0",
-    "typescript": "^3.6.4"
+    "typescript": "^3.9.6"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
Since support for `node@8` was dropped, should the `@babel/preset-env` targets be updated to `{ node: '10' }` as well?

Reply:
Yes, good find.